src/main/java/org/folio/circulation/support/http/client/CqlQuery.java

```diff
@@ -42,6 +42,10 @@
     return of(() -> new CqlQuery(String.format("%s=\"\"", index), none()));
   }
 
+  public static Result<CqlQuery> match(String index, String value) {
```

Comment:
What does match mean? Is it a partial match?

Reply:
Yes, it is match with single `=`.